WordPress/theme-experiments
# Block Themes: Remove theme attributes from block templates

#### Changes proposed in this Pull Request:
Remove theme attributes from block-theme block templates

#### Context:
This PR was created as a follow-up to WordPress/gutenberg#28088. See the aforementioned PR description and code changes for more context.

#### Discussion:

1. What this means for theme designers is that, when creating block templates, the theme attribute can be omitted from block templates (like we've done in this PR). The logic for the Gutenberg full site editor will automatically add a theme attribute that matches the current, activated theme's stylesheet.
2. New sites created for full site editing will _not_ load template parts correctly until https://github.com/WordPress/gutenberg/pull/28088 and https://github.com/WordPress/gutenberg/pull/27910 are added to a release.
3. 
#### Testing:
- Apply this PR
- Run Gutenberg locally (see instructions [here](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/getting-started.md#using-wp-env-to-install-a-local-environment))
- Install the tt1-blocks theme by referencing it in `.wp-env.override.json`
    ```javascript 
      // .wp-env.override.json

      // Replace ~/work/theme-experiments/tt1-blocks with 
      // the path to your locally installed tt1-blocks theme
      {
           "mappings": {
               "wp-content/themes/tt1-blocks": "~/work/themes/tt1-blocks",
           }
      }```
- `npx wp-env start --update`
- Activate the tt1-blocks theme
- Open the front end and the site editor. FSE templates and template parts should load without issue.
- Reinstall the tt1-blocks theme in a subdirectory by modifying `.wp-env.override.json`
   ```javascript 
      // .wp-env.override.json
     
      // Add a test subdirectory in the mapping
      {
           "mappings": {
               "wp-content/themes/test/tt1-blocks": "~/work/themes/tt1-blocks",
           }
      }```
- `npx wp-env start --update`
- Activate tt1-blocks.
- Open the front end and the site editor. FSE templates and template parts should _still_ load without issue.
